guide/daily-spending-wallet/requesting.md

```diff
@@ -34,440 +34,400 @@ Illustration source
-   retina = "/assets/images/guide/daily-spending-wallet/funding/no-inbound@2x.png"
-   layout = "shadow"
-   caption = "Guide users as to how to obtain inbound liquidity so they can receive payments."
-   alt-text = "Screen showing options to receive inbound liquidity."
-   width = 250
-   height = 541
-%}
+If requesting a specific amount denominated in fiat, use a shorter expiry and have the invoice refresh each time it expires. For a wallet designed for in-person payments, a 30 - 60 second expiry works well.
 
-</div>
+If there is no amount defined on the invoice, the invoice is denominated in bitcoin, or the invoice needs to be shared in a message, then use a longer expiry, like 24 hours.
 
-### On-chain
+While it is good to allow users to define their own custom expiries, remember that only advanced users will do this. Choose a sensible default that makes sense for your user-base and how the wallet is intended to be used.
 
-Receiving on-chain involves the sender broadcasting a transaction to the network for it to be confirmed.
+For some background on situations the user might encounter with different invoice expiries, see this [blog post](https://d.elor.me/2022/01/lightning-invoice-expiration-ux-considerations/) from designer Stephen DeLorme.
 
-<div class="center" markdown="1">
+<div class="image-slide-gallery">
 
 {% include picture.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/confirming.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/confirming@2x.png"
-   layout = "float-left-desktop -background -shadow"
-   caption = "For on-chain, keep users updated on incoming pending transactions."
-   alt-text = "Screen showing on-chain transaction being confirmed."
+   image = "/assets/images/guide/daily-spending-wallet/requesting/Fees@2x.png"
+   retina = "/assets/images/guide/daily-spending-wallet/requesting/Fees@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/requesting/Fees@2x.png"
+   layout = "shadow"
+   caption = "Let users know if fees will be charged."
+   alt-text = "Screen showing a payment request with setup fees shown."
    width = 250
    height = 541
+   modalWidth = 250
+   modalHeight = 541
 %}
 
-As on-chain transaction confirmation times vary based on network congestion and fees being paid, it is uncertain exactly when a on-chain payment will be considered complete. Keep users informed of the state of their incoming transactions.
-
-</div>
-
-## Success
-
-Wallet applications should clearly indicate once a payment has been received by the user.
-
-<div class="center" markdown="1">
-
-{% include image.html
-   image = "/assets/images/guide/daily-spending-wallet/requesting/received.png"
-   retina = "/assets/images/guide/daily-spending-wallet/requesting/received@2x.png"
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/requesting/InvoiceExpiration.png"
```

Comment:
Don't think you need a dedicated screen just for this one setting. Since the details screen already shows the expiration of the current invoice, that option could just be clickable to define a custom expiration.